### PR TITLE
Automate: Verify that user can perform wildcard search via APIExport virtual workspace

### PR DESCRIPTION
**Pull request by zxiao-redhat**
This is an automation for test case OCP-59765. 

Test: passed: (31.4s) 2023-02-14T11:45:00 "[area/apiexports] Author:zxiao-Critical-[API] Verify that user can perform wildcard search via APIExport virtual workspace [Suite:kcp/smoke/parallel]"

### CODE REVIEW

File: test/extended/apibinding/apibinding.go [hunk 1]
```diff
@@ -79,4 +79,30 @@ var _ = g.Describe("[area/apiexports]", func() {
...
+	g.It("Author:zxiao-Critical-[API] Verify that user can perform wildcard search via APIExport virtual workspace", func() {
+		g.By("# Create a test workspace")
+		k.SetupWorkSpace()
+
+		g.By("# Create custom resource cowboy using APIResourceSchema")
+		rsTemplate := exutil.FixturePath("testdata", "apibinding", "api_rs.yaml")
+		_, err := exutil.CreateResourceFromTemplate(k, rsTemplate)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("# Export custom resource using APIExport")
```
Comment (kasturinarra) [inline test/extended/apibinding/apibinding.go]: @zimo-xiao should you say "create apiexport for custom resource " instead ?

**Comment by kasturinarra** [comment]
some minor comment, rest looks good to me.

File: test/extended/apibinding/apibinding.go (hunk 1 above)
Comment (zxiao-redhat) [reply test/extended/apibinding/apibinding.go]: Hi @kasturinarra, I have updated my test case. PTAL. Thanks!

**Pull request merged**